dev-docs/bidders/monetix.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: bidder
+title: Monetix
+description: Monetix Adaptor
+biddercode: monetix
+pbjs: true
+pbs: false
+media_types: banner, native, video
+gvl_id: 14 (adkernel)
+tcfeu_supported: true
+gpp_supported: true
```

Comment:
Sorry, but we've deprecated the `gpp_supported` metadata. As described in https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

If you support the IAB’s GPP consent string, add gpp_sids with a comma separated list of section names, e.g. gpp_sids: tcfeu, tcfca, usnat, usstate_all, usp. Default is None.